app/controllers/conversations_controller.rb

```diff
@@ -42,6 +45,14 @@ def destroy
 
   private
 
+  def set_sidebar_conversations
+    @sidebar_conversations = Conversation.grouped_by_increasing_time_interval_for_user(Current.user)
+  end
+
+  def set_sidebar_assistants
+    @sidebar_assistants = Current.user.assistants.to_a - [@conversation&.assistant]
```

Comment:
are you removing the currently active assistant? I think we should keep that one visible in the list and make that the selected one (similar to how you did for conversation). That’s how ChatGPT does it.

Notably, when conversations are selected they are visibly highlighted. When the currently active assistant is selected it’s much more subtle: they just keep the new-pencil icon ever present rather than only appearing on hover.

Reply:
The current assistant is manually displayed at the top. In order to always have it at the top, I didn't want to fiddle with searching the list. Maybe it shouldn't always be at the top? I didn't consider that. Anyway, the logic is in the view and shows the current assistant at the top of the sidebar, in a floating/sticky panel. The rest of the list of assistants is attached to the list of conversations, which is how I remember the OpenAI interface being. I pinged you on the location.

Reply:
Ohh, that seems reasonable enough. We'll try that.

This all looks good, it's just the test failure.

Reply:
I agree with your suggestion. The only real drawback to that approach is that Chat functionally uses the top GPT as their logo on the page. I don't really think that matters as much to this project.